### PR TITLE
Dev mb, last summarization steps w/graph class for blaze v fuseki, more checks&final upload

### PR DESCRIPTION
**Pull request by MBcode**
last summarization steps w/graph class for blaze v fuseki, more checks&final upload

baseurl needed final '/' so now insert gets correct url, and it summarizes to repo.ttl
will do review before final upload, so functionally ready to go,   will still tidy/etc a bit

### CODE REVIEW

**Comment by MBcode** [comment]
I mentioned the docs update, in a docs [issue](https://github.com/earthcube/geocodes/issues/16) comment

File: summarize/README.md [hunk 1]
```diff
@@ -1,23 +1,52 @@
...
+ fuskei
 
-It will call 'fix_runX.sh repo' to get all the rdf and convert it to quads
+ some ec library
 
-then it will call 'summarize_repo.sh repo' to put it in fuseki so it can be queried and summarized
+## INSTALL
 
-producing a: repo.ttl file that can be loaded into the blazegraph summary namespace of your choice
+* install dependinces
```
Comment (valentinedwv) [inline summarize/README.md]: hints. maybe say how to install fuskei?

File: summarize/README.md [hunk 2]
```diff
@@ -1,23 +1,52 @@
...
+ 
+1. if you have not, change to the summarize directory: `cd  earthcube_utilities/summarize`
+2. Insure that a gleaner crawl has already been done, and you have the location of it's bucket
+> gleaner needs to be configured with milled: true 
+
+> put the minio-server url in ECU_S3ADDRESS env var
+
+once gleaner has been run it should have a gleaner/milled path full of repos you can summarize
+
+3. run> `repo2summary.sh repo`
```
Comment (valentinedwv) [inline summarize/README.md]: Do I need to set any env variables. how would i Know unless I read the code

File: summarize/src/fnq.py [hunk 3]
```diff
@@ -67,22 +80,48 @@ def kill_fuseki():
...
 
 
 if __name__ == '__main__':
     import sys
     if(len(sys.argv)>1):
         repo_ = sys.argv[1] #start of if repo was the end of a path
         repo=path_leaf(repo_) #but for now expect repo.nq around so it can make the repo.ttl summary
-        ftsp=os.getenv('fuseki_tmp_summary_port')
+        #ftsp=os.getenv('fuseki_tmp_summary_port')
+        ftsp=os.getenv('tmp_summary_port')
```
Comment (valentinedwv) [inline summarize/src/fnq.py]: ???

File: summarize/src/get_repo.py [hunk 4]
```diff
@@ -21,15 +121,39 @@ def get_repo(repo):
         os.chdir(repo)
         cwd=os.getcwd()
         print(f'cd to: cwd={cwd} to get the repo={repo} to downloads files into')
-    ec.wget_oss_repo(repo) #defaults to bucket=ncsa_minio =https://oss.geocodes.ncsa.illinois.edu/
+    #ec.wget_oss_repo(repo) #defaults to bucket=ncsa_minio =https://oss.geocodes.ncsa.illinois.edu/
+    #ec.wget_oss_repo(repo=repo,path="gleaner/milled",bucket=default_bucket)
+    wget_oss_repo(repo=repo,path="gleaner/milled",s3address=default_s3address)
```
Comment (valentinedwv) [inline summarize/src/get_repo.py]: wget?

File: summarize/src/get_repo.py [hunk 5]
```diff
@@ -21,15 +121,39 @@ def get_repo(repo):
...
+    else:
+        return url
 
+#later can import click, have 2nd arg be named eg.-bucket, and incl a -help
 if __name__ == '__main__':
     import sys
+    print(f'argv={sys.argv}')
+    default_s3address= "https://oss.geocodes.ncsa.illinois.edu/"
+    #s3=os.getenv("S3ADDRESS") 
+    s3=os.getenv("ECU_S3ADDRESS")
```
Comment (valentinedwv) [inline summarize/src/get_repo.py]: Needs to be in doco

**Review by valentinedwv — CHANGES_REQUESTED** [review]
Update the documentation to tell me what env varaiables i'm supposed to be using.
Perhaps use arparse on getrepo, so that we don't have to use them.

File: summarize/README.md (hunk 1 above)
Comment (MBcode) [reply summarize/README.md]: get_repo.py reads ECU_S3ADDRESS and an env var, but will add argparse as well
Once you switched to blaze (that is already installed) we don't use fuseki anymore

Comment (MBcode) [reply summarize/README.md]: updated dependency section, and readme already asks for ECU_S3ADDRESS if you want a different one;  the help for get_repo.py will also allow overriding that

File: summarize/src/fnq.py (hunk 3 above)
Comment (MBcode) [reply summarize/src/fnq.py]: i'll close out this outdated comment too; as no fuseki being used now the fnq.py fuseki started is only around as a util, for those that want quick local access

File: summarize/src/get_repo.py (hunk 4 above)
Comment (MBcode) [reply summarize/src/get_repo.py]: if this isn't part of stock alpine in the container, I can move to requesting them

File: summarize/src/get_repo.py (hunk 5 above)
Comment (MBcode) [reply summarize/src/get_repo.py]: another outdated comment/but think env var has been in the [docs](https://github.com/earthcube/earthcube_utilities/blob/dev_mb/summarize/README.md) for awhile now

File: summarize/README.md (hunk 2 above)
Comment (MBcode) [reply summarize/README.md]: old comment/bc in [doc](https://github.com/earthcube/earthcube_utilities/blob/dev_mb/summarize/README.md)s for a bit

File: summarize/src/get_repo.py (hunk 4 above)
Comment (MBcode) [reply summarize/src/get_repo.py]: changed to request, and updated doc

Comment (MBcode) [reply summarize/src/get_repo.py]: was verified, put requests back in requirements.txt should be ok, but getting:
'does not appear to be a Python project: no pyproject.toml' again

**Review by valentinedwv — APPROVED** [review]
I'd say remove dead code, but works so approved

File: summarize/src/get_repo.py (hunk 4 above)
Comment (MBcode) [reply summarize/src/get_repo.py]: will get commented out of workflow

**Pull request merged**